[2024-08-16T22:18:42.838+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-16T22:18:42.848+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: predict_values_TS_7.get_datasets_from_google scheduled__2024-08-04T00:00:00+00:00 [queued]>
[2024-08-16T22:18:42.852+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: predict_values_TS_7.get_datasets_from_google scheduled__2024-08-04T00:00:00+00:00 [queued]>
[2024-08-16T22:18:42.852+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-16T22:18:42.862+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_datasets_from_google> on 2024-08-04 00:00:00+00:00
[2024-08-16T22:18:42.866+0000] {warnings.py:110} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4169) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-16T22:18:42.867+0000] {standard_task_runner.py:63} INFO - Started process 4174 to run task
[2024-08-16T22:18:42.867+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'predict_values_TS_7', 'get_datasets_from_google', 'scheduled__2024-08-04T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpw9rnpq9o']
[2024-08-16T22:18:42.869+0000] {standard_task_runner.py:91} INFO - Job 16: Subtask get_datasets_from_google
[2024-08-16T22:18:42.903+0000] {task_command.py:426} INFO - Running <TaskInstance: predict_values_TS_7.get_datasets_from_google scheduled__2024-08-04T00:00:00+00:00 [running]> on host 08898c2f83c5
[2024-08-16T22:18:42.959+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ruslan Solarev' AIRFLOW_CTX_DAG_ID='predict_values_TS_7' AIRFLOW_CTX_TASK_ID='get_datasets_from_google' AIRFLOW_CTX_EXECUTION_DATE='2024-08-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-04T00:00:00+00:00'
[2024-08-16T22:18:42.960+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-16T22:18:57.838+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-16T22:18:57.838+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-16T22:18:57.844+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=predict_values_TS_7, task_id=get_datasets_from_google, run_id=scheduled__2024-08-04T00:00:00+00:00, execution_date=20240804T000000, start_date=20240816T221842, end_date=20240816T221857
[2024-08-16T22:18:57.873+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-16T22:18:57.885+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-16T22:18:57.888+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
